packages/filecoin-message/src/index.ts

```diff
@@ -25,7 +25,15 @@ export interface LotusMessage {
   GasLimit: number
   GasFeeCap: string
   Method: number
-  Params?: string | string[] | undefined
+  Params?: string | string[] | null
```

Comment:
2 things here:

1. I haven't tested passing `null` as params explicitly to a node, I'm not sure if this will fly. I need to spin up a node to test this, but the curl request to make would look something like:

```curl
curl --location --request POST 'http://127.0.0.1:1234/rpc/v0' \
--header 'Content-Type: application/javascript' \
--header 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJyZWFkIiwid3JpdGUiLCJzaWduIiwiYWRtaW4iXX0.GRFjz-pVUeD4GRzHFFvr2ABWkMoaYdHY8Mq4L2jpZeQ' \
--data-raw '{
"jsonrpc": "2.0",
"method": "Filecoin.WalletSignMessage",
"id": 1,
"params": ["fromAddress", {
	"To": "f1hvuzpfdycc6z6mjgbiyaiojikd6wk2vwy7muuei",
	"From": "fromAddress",
	"Nonce": 0,
	"Value": "1000",
	"Method": 0,
	"GasPremium": "0",
	"GasLimit": 0,
        "GasFeeCap": "0"
	"Params": null
}]
}'
```

2. I'm realizing now that this `params` can actually be:

`string | any[]` (in addition to potentially `null`).

I know you didn't change to `string[]` but we might as well make this change here anyways.

Reply:
True.